ngrinder-controller/src/main/java/org/ngrinder/starter/NGrinderControllerStarter.java

```diff
@@ -307,11 +264,38 @@ public static void main(String[] args) throws Exception {
 		final ClusterMode clusterMode = ClusterMode.valueOf(server.clusterMode);
 		clusterMode.parseArgs(unknownOptions.toArray(new String[unknownOptions.size()]));
 		System.getProperties().putAll(server.params);
-		server.run();
+		SpringApplication.run(NGrinderControllerStarter.class, args);
+		cleanupPreviouslyUnpackedFolders();
```

Comment:
Is it safe to run ```cleanupPreviouslyUnpackedFolders``` after Spring shutdown?
I think there is a lot of case the ngrinder is killed using ```kill -9```
Therefore this clean up process will not be executed.

Reply:
This process is run at Spring startup not after shutdown. 

so, when startup server It will clean up previous unpacked files.

Reply:
This process is run at spring startup, not after shutdown.
So, It will clean up previous unpacked files.

Reply:
Nop. It seems like this is executed when shutdown.. 
Because SpringApplication.run is not stopped before the process is going to down.
Please check it.

Reply:
yes i will check it.